src/editor/lib/Schema.js

```diff
@@ -560,7 +570,26 @@ var SchemaAttribute = function (node, schema) {
    * @var array
    */
   var documentationCache = undefined;
-   
+  
+  /**
+   * Transform documentation text to link to the online documentation when it
+   * contains a reference.
+   * 
+   * @return string The transformed input string.
+   */
+  function createDocumentationWebLinks( text )
+  {
+    return text.replace(new RegExp( ":ref:[`'](.+?)[`']", 'g'), function(match, contents){
+      var 
+        reference = contents.match( /^(.*?) *<([^ ]*)>$/ ),
```

Comment:
Iḿ not really sure, but spaces might be allowed in the <>, so maybe the regex should allow anything but > like: ` /^(.*?) *<([^>]*)>$/`. What do you think?